test/e2e/specs/editor/blocks/image.spec.js

```diff
@@ -516,69 +516,6 @@ test.describe( 'Image', () => {
 		}, 'should replace the original image with the second image' ).toPass();
 	} );
 
-	test( 'should allow dragging and dropping HTML to media placeholder', async ( {
```

Comment:
Decided to remove this test, it's basically testing the same thing as `can be replaced by dragging-and-dropping images from the inserter` (See #67317), but tries to emulate the whole drag process. 

cc @ellatrix, @tellthemachines